reference/carts.v3.yml

```
@@ -1,4 +1,4 @@
-openapi: '3.0.3'
+openapi: 3.0.3
```

```suggestion
openapi: '3.0.3'
```

string type per oas spec